order-service/pom.xml

```diff
@@ -251,18 +251,11 @@
                 <configuration>
                     <image>
                         <name>dockertmt/mmv2-${project.artifactId}:${project.version}</name>
```

Comment:
@coderabbitai, can you please let me know the best way to solve the issue of adding libstdc++6 to the image that is generated by spring-boot:build-image